include/neighbor.h

```diff
@@ -21,8 +21,10 @@ namespace diskann {
     }
 
     inline bool operator<(const Neighbor &other) const {
```

Comment:
Should we do the same change for `SimpleNeighbor` class below?

Reply:
yes, we can add it too. I saw it is referenced in some test code and then only referenced once in source code below:
https://github.com/microsoft/DiskANN/blob/2e1e328e9362196de70d9f25ca7fc542b9438d5c/src/pq_flash_index.cpp#L924
And the variable `medoid_dists` is not referenced anywhere, we can remove that line @harsha-simhadri ?

Reply:
Is this related to this PR? Or a separate issue?

Reply:
Probably not related. So you suggest not to touch it?